src/views/ContinuationOut.vue

```diff
@@ -97,6 +97,8 @@
                         v-model="detailComment"
                         placeholder="Add a Detail that will appear on the ledger for this entity."
                         :maxLength="maxDetailCommentLength"
+                        :isRoleStaff="isRoleStaff"
+                        :isContinuationOutDetailComment="true"
```

Comment:
Why do you need these extra properties, if this entire section (code block) is only displayed for staff?

Reply:
DetailComment component is used elsewhere. I thought since it only applies to ContOut, then I need to specify it. 

I'll update this

Reply:
In the future, if this kind of thing is needed, it's better for common components to know as little as possible about the world around them (which them more portable).

Specifically, it would be better if this component doesn't know what type of filing it's working on. What you could do instead is create a generic prop that, in this case, would tell the component to override the rules.

Anyway, I think none of this is needed in this PR so FYI only :)